src/tests/ftest/pool/multiple_creates_test.py

```diff
@@ -250,27 +271,18 @@ def test_create_three(self):
                 self.fail("Pool {0} not found on host {1}.\n".format(uuid_str_3,
                                                                      host))
 
-            delete_cmd_1 = (
-                "{0} destroy-pool "
-                "-i {1} "
-                "-s {2} "
-                "-f".format(self.daosctl, uuid_str_1, setid))
-
-            delete_cmd_2 = (
-                "{0} destroy-pool "
-                "-i {1} "
-                "-s {2} "
-                "-f".format(self.daosctl, uuid_str_2, setid))
-
-            delete_cmd_3 = (
-                "{0} destroy-pool "
-                "-i {1} "
-                "-s {2} "
-                "-f".format(self.daosctl, uuid_str_3, setid))
-
-            process.system(delete_cmd_1)
-            process.system(delete_cmd_2)
-            process.system(delete_cmd_3)
+            result = dmg.pool_destroy(pool=uuid_str_1)
+            if "failed" in result.stdout:
+                self.log.info("Unable to destroy pool %s", uuid_str_1)
```

Comment:
Are we expecting the destroy to succeed? If so, we should fail the test instead of writing the log.

Reply:
Done. Thank you!